shell/common/shell.cc

```diff
@@ -36,20 +36,16 @@ namespace flutter {
 constexpr char kSkiaChannel[] = "flutter/skia";
 
 std::unique_ptr<Shell> Shell::CreateShellOnPlatformThread(
-    DartVMRef vm,
     TaskRunners task_runners,
     Settings settings,
-    fml::RefPtr<const DartSnapshot> isolate_snapshot,
-    fml::RefPtr<const DartSnapshot> shared_snapshot,
     Shell::CreateCallback<PlatformView> on_create_platform_view,
     Shell::CreateCallback<Rasterizer> on_create_rasterizer) {
   if (!task_runners.IsValid()) {
     FML_LOG(ERROR) << "Task runners to run the shell were invalid.";
     return nullptr;
   }
 
-  auto shell =
-      std::unique_ptr<Shell>(new Shell(std::move(vm), task_runners, settings));
+  auto shell = std::unique_ptr<Shell>(new Shell(task_runners, settings));
```

Comment:
I think we need to tell the shell about how to get a VM at this point.  The platform view may need it for some embedding implementations.

Reply:
Also nit: this can just be `auto shell = std::make_unique<Shell>(task_runners, settings);`